actions/global_actions.jsx

```diff
@@ -236,7 +236,7 @@ export function emitUserLoggedOutEvent(redirectTo = '/', shouldSignalLogout = tr
     // If the logout was intentional, discard knowledge about having previously been logged in.
     // This bit is otherwise used to detect session expirations on the login page.
     if (userAction) {
-        LocalStorageStore.setWasLoggedIn(false);
+        LocalStorageStore.setWasLoggedIn(window.basename, false);
```

Comment:
Is there a way we could leverage `getBasePath` from `selectors/general.js` as much as possible, just to avoid using `window.basename` directly unless absolutely necessary?